src/app/client/src/app/modules/program-dashboard/components/program-datasets/program-datasets.component.html

```diff
@@ -9,7 +9,7 @@
        <div class="mr-12">
         <button type="button" [ngClass]="layoutConfiguration ? 'sb-btn-round' : ''"
       class="sb-btn sb-btn-normal sb-btn-link sb-btn-link-primary sb-left-icon-btn px-0" tabindex="0"
-      (click)="goBack()" id="goBack" attr.aria-label="{{resourceService?.frmelmnts?.btn?.back}}">
+      (click)="closeDashboard()" id="goBack" attr.aria-label="{{resourceService?.frmelmnts?.btn?.back}}">
```

Comment:
why function name change?

Reply:
A new function has been added specific for going back to previous location and closing the dashboard previous function goBack is related to filter pop-up closing functionality based on few scenarios